fiat-core/src/main/java/com/netflix/spinnaker/fiat/model/UserPermission.java

```diff
@@ -46,7 +47,11 @@ public View getView() {
   @Data
   public class View {
     String name = UserPermission.this.id;
-    Map<String, Object> resources = ImmutableMap.of(
+    Set<ServiceAccount.View> serviceAccounts = UserPermission.this.serviceAccounts
+        .stream()
+        .map(ServiceAccount::getView)
+        .collect(Collectors.toSet());
+    Map<String, Object> cloudResources = ImmutableMap.of(
```

Comment:
`View` classes are used solely for serialization, so I think the typing is far less of a concern.